server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -255,6 +255,8 @@ boolean inSafeMode() {
           refs =
               Stream.concat(refs, Stream.of(new Reference(tm.getTableId(), tm.getDirName(), true)));
         }
+        refs = Stream.concat(refs, getContext().getAmple().getScanServerFileReferences()
```

Comment:
I think this would concat the scan servers refs for each tablet, but not completely sure.  I may have fixed this in e33ec17237c758f47bbcc25accf2dfa37b38617a